### PR TITLE
feat: add proving deadline option view live sectors information

### PR DESCRIPTION
**Pull request by beck-8**
## Checklist

Before you mark the PR ready for review, please make sure that:
- [x] All commits have a clear commit message.
- [x] The PR title is in the form of of `<PR type>: <area>: <change being made>`
    - example: ` fix: mempool: Introduce a cache for valid signatures`
    - `PR type`: _fix_, _feat_, _INTERFACE BREAKING CHANGE_, _CONSENSUS BREAKING_, _build_, _chore_, _ci_, _docs_,_perf_, _refactor_, _revert_, _style_, _test_
    - `area`: _api_, _chain_, _state_, _vm_, _data transfer_, _market_, _mempool_, _message_, _block production_, _multisig_, _networking_, _paychan_, _proving_, _sealing_, _wallet_, _deps_
- [x] This PR has tests for new functionality or change in behaviour
- [x] If new user-facing features are introduced, clear usage guidelines and / or documentation updates should be included in https://lotus.filecoin.io or [Discussion Tutorials.](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)
- [x] CI is green

### CODE REVIEW

**Comment by jennijuju** [comment]
> If new user-facing features are introduced, clear usage guidelines and / or documentation updates should be included in https://lotus.filecoin.io/

thanks for contributing - could you please also add user documentation as suggested in the PR template?

**Comment by beck-8** [comment]
> > If new user-facing features are introduced, clear usage guidelines and / or documentation updates should be included in https://lotus.filecoin.io/
> 
> thanks for contributing - could you please also add user documentation as suggested in the PR template?

yes,I will write user documentation later in [Discussion Tutorials](https://github.com/filecoin-project/lotus/discussions/categories/tutorials)

**Comment by beck-8** [comment]
> > If new user-facing features are introduced, clear usage guidelines and / or documentation updates should be included in https://lotus.filecoin.io/
> 
> thanks for contributing - could you please also add user documentation as suggested in the PR template?

user documentation already added @jennijuju 
https://github.com/filecoin-project/lotus/discussions/8723

File: cmd/lotus-miner/proving.go [hunk 1]
```diff
@@ -320,33 +361,64 @@ var provingDeadlineInfoCmd = &cli.Command{
...
+				if err != nil {
+					return err
+				}
+
+				fmt.Printf("Partition Index:          %d\n", pIdx)
+				fmt.Printf("Sectors:                  %d\n", sectorCount)
+				fmt.Printf("Sector Numbers:           %v\n", sectorNumbers)
+				fmt.Printf("Faults:                   %d\n", faultsCount)
+				fmt.Printf("Faulty Sectors:           %d\n", fn)
+			}
```
[the remark this answers is not in the document]
Comment (beck-8) [reply cmd/lotus-miner/proving.go]: @geoff-vball  Sorry, I originally generated redundant code in order to reduce the number of judgments, and it has been corrected.

**Pull request merged**